.github/workflows/ci-benchmark.yml

```diff
@@ -30,25 +30,16 @@ permissions:
   contents: write
 
 jobs:
-  build-all:
```

Comment:
Could you re-add this section?

Reply:
Tbh I don't understand why we need this section. Do you mind helping understand?

Reply:
I see, thanks for the explanation! I should also make `benchmark` job dependent on this `build-all` job, right?

Reply:
Correct